From 610009806fa734dbef0a3dfa5ed68839b2627419 Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Thu, 11 Jul 2024 11:03:49 +0200
Subject: [PATCH] Update setup.py

---
 setup.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index 2d9c4c7..eaea370 100644
--- a/setup.py
+++ b/setup.py
@@ -15,11 +15,11 @@
     author='Anton Bochkarev',
     author_email='',
     description='',
-    python_requires='<3.11',
+    python_requires='<3.13',
     install_requires=
     [
     'scipy',
-    'tensorflow<=2.12.1',
+    'tensorflow',
     'numpy',
     'pandas',
     'ase'
